liquibase-standard/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -221,6 +224,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .setDescription("Will perform xsd validation of XML changelog files. When many XML changelog files are included this validation may impact Liquibase performance. Defaults to true.")
                 .setDefaultValue(true)
                 .build();
+
+        UI_SERVICE = builder.define("uiService", UIServiceEnum.class)
+                .setDescription("Changes the default UI Service Logger used by Liquibase. Options are CONSOLE or LOGGER.")
+                .setDefaultValue(UIServiceEnum.CONSOLE)
+                .build();
```

Comment:
One more thing here, I recommend you use the value handler util.

For an example, search for `ValueHandlerUtil.getEnum`.

Long term the command framework should automatically do this when using an enum.

Reply:
that's interesting - I searched for it and found out that it only exists in tests (?) - seems it was used at UpdateCommandStep but was removed. Anyway here is 
 it:  https://github.com/liquibase/liquibase/pull/4584/commits/718d57d3e6ebd22cacf63022bb275c66bdae9592 .